plugins/modules/content_upload.py

```diff
@@ -128,6 +139,7 @@ def main():
             src=dict(required=True, type='path', aliases=['file']),
             repository=dict(required=True, type='entity', scope=['product'], thin=False),
             product=dict(required=True, type='entity', scope=['organization']),
+            ostree_repository_name=dict(required=False, type='str'),
```

Comment:
You'll have to add `ostree_repository_name` to the `DOCUMENTATION` section above.